spec/game_spec.js

```diff
@@ -0,0 +1,69 @@
+describe('Game', function() {
```

Comment:
The code is easy to follow and is well named so that it is readable. Frame and Game are nicely separated with good encapsulation. 
It would be useful to remove some of the commented out code (and the console.log statements) when uploading to Git to make the code a bit clearer to read